arch/arc/boot/dts/haps_hs5x.dts

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (C) 2023 Synopsys, Inc. (www.synopsys.com)
+ */
+/dts-v1/;
+
+/include/ "skeleton_hs.dtsi"
+/include/ "skeleton_haps.dtsi"
+
+/ {
+	model = "snps,zebu_hs5x";
+	compatible = "snps,zebu_hs";
+
+	memory {
+		device_type = "memory";
```

Comment:
Note that technically you don't need to have device type overwrite here - as it's same as in the entry you update.

Reply:
It makes sense. I see arm .dts also drop this. I will drop it.